cli/jobs/automl-standalone-jobs/cli-automl-image-classification-multiclass-task-fridge-items/cli-automl-image-classification-multiclass-task-fridge-items-automode.yml

```diff
@@ -0,0 +1,25 @@
+$schema: https://azuremlsdk2.blob.core.windows.net/preview/0.0.1/autoMLJob.schema.json
+
+type: automl
+
+experiment_name: dpv2-cli-automl-image-classification-experiment
+description: A multi-class Image classification job using fridge items dataset
+
+compute: nc6
```

Comment:
Minor: Should we change this to "azureml:gpu-cluster" since it is mentioned in the .txt file

Reply:
Yes, thank you for catching!